.github/workflows/go-cross.yml

```diff
@@ -13,6 +13,7 @@ concurrency:
 
 jobs:
   build:
+    if: github.event.pull_request.draft == false
```

Comment:
Seems like there is a better way

```
on:
  pull_request:
    types: [review_requested, ready_for_review]
```

- https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull-request-event-pull_request
- https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onevent_nametypes